core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -543,8 +543,9 @@ oppia.filter('summarizeAnswerGroup', [
 
       if (answerGroup.rules) {
         var firstRule = $filter('convertToPlainText')(
+          $filter('formatRtePreview')(
```

Comment:
We should apply the filter as narrowly as possible. Presumably the Rte preview filter should only be applied to HTML fields -- so shouldn't it be applied to the elements of answerChoices instead?